baseapp/baseapp.go

```diff
@@ -237,7 +237,10 @@ func (app *BaseApp) NewContext(isCheckTx bool, header abci.Header) sdk.Context {
 	if isCheckTx {
 		return sdk.NewContext(app.checkState.ms, header, true, nil, app.Logger)
 	}
-	return sdk.NewContext(app.deliverState.ms, header, false, nil, app.Logger)
+	aa := app.deliverState
```

Comment:
This style strikes me as a bit unusual - if you want more clarity, why not use descriptive variable names? Otherwise we can pass all the arguments inline to `sdk.NewContext`

Reply:
sorry this was for debugging - reversing